scripts/apps/contacts/components/fields/ContactName.tsx

```diff
@@ -1,8 +1,12 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+const notEmpty = (str) => typeof str === 'string' && str.length > 0;
+
 export const ContactName: React.StatelessComponent<any> = ({item}) => {
-    const displayContact = item.first_name ? `${item.first_name} ${item.last_name}` : item.organisation;
+    const displayContact = notEmpty(item.first_name) || notEmpty(item.last_name) ?
```

Comment:
Can be: 

> const displayContact = [item.first_name, item.last_name].filter(Boolean).join(' ') || item.organisation;

?

Reply:
I prefer it to be explicit, and not rely on type castings. When you filter using `Boolean`, the result is the same, but it's not obvious what you are doing. It would be fine either way in this case, but I like keeping it simple as a general practice in the codebase.

Reply:
The functional programming is very powerful but sometime not so easy to understand. To me looks that here is unnecessary code but it is your choice, this is something minor.